CHANGES.rst

```diff
@@ -10,6 +10,188 @@
 
 .. towncrier release notes start
 
+3.9.2 (2024-01-28)
+==================
+
+Bug fixes
+---------
+
+- Fixed websocket connection leak.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`7978`.
+
+
+
+- Fixed `web.FileResponse` doing blocking I/O in the event loop.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8012`.
+
+
+
+- Fixed double compress when compression enabled and compressed file exists.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8014`.
+
+
+
+- Added runtime type check for ``ClientSession`` ``timeout`` parameter.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8021`.
+
+
+
+- Fixed an unhandled exception in the Python HTTP parser on header lines starting with a colon -- by :user:`pajod`.
+
+  Invalid request lines with anything but a dot between the HTTP major and minor version are now rejected.
+  Invalid header field names containing question mark or slash are now rejected.
+  Such requests are incompatible with :rfc:`9110#section-5.6.2` and are not known to be of any legitimate use.
+
+  (BACKWARD INCOMPATIBLE)
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8074`.
+
+
+
+- Improved validation of paths for static resources -- by :user:`bdraco`.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8079`.
+
+
+
+
+Features
+--------
+
+- Added support for passing :py:data:`True` to ``ssl`` while deprecating :py:data:`None` -- by :user:`xiangyan99`.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`7698`.
+
+
+
+
+Improved documentation
+----------------------
+
+- Updated :ref:`contributing/Tests coverage <aiohttp-contributing>` section to show how we use ``codecov`` -- by :user:`Dreamsorcerer`.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`7916`.
```

Comment:
I'd move this fragment to the `contrib` section since it doesn't really concern the end-users, but contributors to the project.